.travis.yml

```diff
@@ -85,7 +91,6 @@ env:
 before_install:
  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update ; fi
  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install zsh                             ; fi
- - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install cmake                           ; fi
  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install boost@1.55                      ; fi
  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install gmp                             ; fi
  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install mpfr                            ; fi
```

Comment:
This could probably be factorized into:
```shell
if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update && brew install zsh boost@1.55 gmp mpfr; fi
```

If I remember well, I think that the syntax `[[ ]]`` and `==` are two bash extensions, and that the POSIX-shell syntax is:
```shell
[ "$TRAVIS_OS_NAME" = "osx" ]
```